00-programming-fundamentals/03-installfest.md

```diff
@@ -13,6 +13,9 @@ Xcode is an **integrated development environment** (often called an **IDE**) for
 ### Trust but Verify
 Run `$ xcode-select -p`. If it tells you `/Library/Developer/CommandLineTools`, you're ready to move on.
 
+If the Xcode Command-Line tools are not installed you can install it with:
+`xcode-select -p` and be patient because it will take time to install.
```

Comment:
The paragraph above this mentions running `xcode-select -p` already.

Will students need to run the command twice to install the command-line tools? If so we might want to make that more explicit.